jmclient/jmclient/cryptoengine.py

```diff
@@ -3,16 +3,17 @@
 
 from binascii import hexlify, unhexlify
 from collections import OrderedDict
+import struct
 
 
 from . import btc
 from .configure import get_network
 
 
-TYPE_P2PKH, TYPE_P2SH_P2WPKH, TYPE_P2WPKH = range(3)
+TYPE_P2PKH, TYPE_P2SH_P2WPKH, TYPE_P2WPKH = [b'\x00', b'\x01', b'\x02']
 NET_MAINNET, NET_TESTNET = range(2)
 NET_MAP = {'mainnet': NET_MAINNET, 'testnet': NET_TESTNET}
-WIF_PREFIX_MAP = {'mainnet': 0x80, 'testnet': 0xef}
+WIF_PREFIX_MAP = {'mainnet': b'\x80', 'testnet': b'\xef'}
```

Comment:
does it make sense to change this to bytes only to later convert the byte to int again?

Reply:
It's mostly for consistency with the changes in `jmbitcoin`.